other/upgrade_2-1_mysql.sql

```diff
@@ -1022,22 +1022,22 @@ if (!empty($select_columns))
 		while ($row = $smcFunc['db_fetch_assoc']($request))
 		{
 			if (!empty($row['aim']))
-				$inserts[] = array($row['id_member'], -1, 'cust_aolins', $row['aim']);
+				$inserts[] = array($row['id_member'], 0, 'cust_aolins', $row['aim']);
```

Comment:
https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/Sources/Load.php#L1753

Probably should be `1`